jest.config.js

```diff
@@ -1,6 +1,6 @@
 /*
```

Comment:
Does this work as a global config? Could we remove all the `jest.config.js` files?

Reply:
No, every package needs a own config, but we can let jest life in the root and have a common config we extend in the children